README.md

```diff
@@ -1,2 +1,44 @@
-# Mobile-first-D3
-Mobile First approach
+# PORTFOLIO - MICROVERSE TASK
+
+> This a "mobile-first" approach, that gives more priorities in developing mobile version first and perfecting the mobile user experience. Due to the fact that there are rising numbers of mobile users as compared to the desktop users.
+
+## Built With
+
+- Html,Css,
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+All you need VScompiler or a compiler of your choice.
+
+### Setup
+
+You will need to clone the repo and open the html with live server.
```

Comment:
- Let's also add some commands on how the user can clone the repo to make our readme more professional.